<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArduinoWindow</class>
 <widget class="QDialog" name="ArduinoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arduino programming</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0" columnstretch="0,0">
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTextEdit" name="logText">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
